project_hr/__init__.py

```diff
@@ -1,3 +1,3 @@
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
Please keep the diff minimal and avoid this change, which adds no real value. The previous link works perfectly fine.

The same across all files.